### ISSUE TITLE
Docs: Blog post contains confusing phrasing around package dist path updates

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/typescript-eslint#7286]
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

Reported by @Zamiell in the Discord: https://discord.com/channels/1026804805894672454/1084238921677946992/1131985918346154136

> I'm reading the v6 blog post here: https://typescript-eslint.io/blog/announcing-typescript-eslint-v6
> It says that people using TSESLint need to change their imports to this:
> ```ts
> import * as TSESLint from '@typescript-eslint/ts-eslint';
> ```
>
> Is this a typo? @typescript-eslint/ts-eslint is not a package that exists on npm

Technically the blog post is just surfacing that as a hypothetical path... but it's kind of confusingly phrased. I can definitely see how one would interpret it as putting forward an incorrect package path. It'd be better to either suggest an actual import path & specifiers folks would be using in the wild, or use a placeholder like `...`.

### Affected URL(s)

https://typescript-eslint.io/blog/announcing-typescript-eslint-v6/#package-exports

**Comment by bradzacher** [comment]
> Technically the blog post is just surfacing that as a hypothetical path... but it's kind of confusingly phrased

If the blog post is saying that then it's wrong - it should be `@typescript-eslint/utils/ts-eslint`
